packages/js/src/settings/routes/templates/taxonomy.js

```diff
@@ -26,7 +27,8 @@ const FormikReplacementVariableEditorFieldWithDummy = withFormikDummyField( Form
  * @param {string[]} postTypes The connected post types.
  * @returns {JSX.Element} The taxonomy element.
  */
-const Taxonomy = ( { name, label, postTypes: postTypeNames } ) => {
+// eslint-disable-next-line complexity
```

Comment:
This line makes it so the JSDoc detection is off, hence you disabled that?
I'd recommend disabling the complexity for the whole file instead of the JSDoc, it seems that is more what you wanted to achieve here.

Reply:
fixed